### ISSUE TITLE
pip install poetry- is it needed?

### ISSUE THREAD

**Issue by scout-mckee** [UBC-MDS/datexplore#96]
just noticed one thing- I don't think we need to include installing poetry because once our package is hosted on PyPI pip will install the entire package. I might be missing something though so just let me know!:) thanks!

**Comment by killerninja8** [comment]
I'm not too sure, but the group I reviewed did not include pip install poetry in their instructions so I had to do it manually.

**Comment by killerninja8** [comment]
I can try doing this without pip install on someone else's environment (without poetry installed) to see how it works. Will report back when I get an update.

**Comment by scout-mckee** [comment]
yeah I get that but I think that is because it is not until the end of this milestone that it is hosted on pypi. Like I think once it is up there we can take away the poetry install because the our package will be available from pypi

**Comment by scout-mckee** [comment]
> I can try doing this without pip install on someone else's environment (without poetry installed) to see how it works. Will report back when I get an update.

Wait until we get ours up and running on pypi first

**Comment by killerninja8** [comment]
We I think that's a great idea. Let's delete it after it is on PyPI.
